src/main/steps/applicant1/address/confirm-change/post.ts

```diff
@@ -0,0 +1,33 @@
+import autobind from 'autobind-decorator';
```

Comment:
@pankajhmcts Please verify the approach

@ShakiraKP
Recommended to use to-api-format.ts for such prefix mapping instead of creating new controllers